### PR TITLE
Minor clarifications & corrections in tutorial text.

### PR DESCRIPTION
**Pull request by dbc**
A few text clarifications to quickstart_in_rvis and move_group_python_interface tutorials.  No code changes.

### CODE REVIEW

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py [hunk 1]
```diff
@@ -230,6 +230,7 @@ def plan_cartesian_path(self, scale=1):
     ## ^^^^^^^^^^^^^^^
     ## You can plan a Cartesian path directly by specifying a list of waypoints
     ## for the end-effector to go through:
+    ## (If executing  interactively in a Python shell, set scale = 1.0).
```
Comment (davetcoleman) [inline doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: No need for parenthesis here, just replace ":" with "."

File: doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst [hunk 2]
```diff
@@ -48,6 +48,7 @@ Step 1: Launch the Demo and Configure the Plugin
   * Make sure the **Robot Description** field is set to ``robot_description``.
 
   * Make sure the **Planning Scene Topic** field is set to ``/planning_scene``.
+    (Click on topic name to expose topic-name drop-down.)
```
Comment (davetcoleman) [inline doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst]: No need for parenthesis here

File: doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py (hunk 1 above)
Comment (dbc) [reply doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py]: Updated.

File: doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst (hunk 2 above)
Comment (dbc) [reply doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst]: Missed this comment the first time.  Pushed again.

**Pull request merged**